core/commands/pin/remotepin.go

```diff
@@ -480,8 +480,10 @@ var addRemotePinServiceCmd = &cmds.Command{
 			cfg.Pinning.RemoteServices = map[string]config.RemotePinningService{}
 		}
 		cfg.Pinning.RemoteServices[name] = config.RemotePinningService{
-			ApiEndpoint: url,
-			ApiKey:      key,
+			Api: config.RemotePinningServiceApi{
+				ApiEndpoint: url,
+				ApiKey:      key,
+			},
```

Comment:
The config layout for service.Api is changed in this PR, it should probably be in the other one (and this one rebased)